src/common/options/global.yaml.in

```diff
@@ -4143,6 +4143,7 @@ options:
   - bitmap
   - stupid
   - avl
+  - btree
```

Comment:
hi @ifed01 , thank you for your reply, BtreeAllocator support for bluefs_allocator has been added here, are there any other problems? Looking forward to your reply.

Reply:
Looks good to me now - will approve once all the basic checks below are done

Reply:
That sounds pretty good

Reply:
> Looks good to me now - will approve once all the basic checks below are done
That sounds pretty good